testsuite/classlibrary/TestServer_clientID_booted.sc

```diff
@@ -70,4 +70,152 @@ TestServer_clientID_booted : UnitTest {
 		s.quit;
 		s.remove;
 	}
+
+	// test that repeated login from the same server object to a running server process
+	// ends with correct server status, and keeps clientID and allocators  as is.
+	test_repeatedLogin {
+		var server, cond, timer, lastNodeID;
+
+		server = Server(thisMethod.name.asSymbol);
+		server.options.maxLogins = 4;
+		server.clientID = 3;
+		this.bootServer(server);
+
+		// use some nodeIDs so we know whether nodeID allocator is reset
+		10.do { server.nextNodeID };
+		lastNodeID = server.nextNodeID;
+
+		// simulate losing network contact by turning off watcher,
+		// but actually leave the server process running
+		server.statusWatcher
+		.stopStatusWatcher
+		.stopAliveThread
+		.notified_(false)
+		.serverRunning_(false);
+
+		//		1.wait;
```

Comment:
Can you please remove this? Everything else looks good, thanks for the tests!

Reply:
sure, done!